stories/index.js

```diff
@@ -41,7 +41,47 @@ storiesOf('Picker', module)
       showPreview={boolean('Show preview', true)}
       showSkinTones={boolean('Show skin tones', true)}
       custom={CUSTOM_EMOJIS}
-    />
+    />))
+    .add('not-found-img', () => (
```

Comment:
More idiomatic for [storybook](https://github.com/storybooks/storybook) would be 'with a custom not found Image' and 'with a custom not found SVG' as the story name.